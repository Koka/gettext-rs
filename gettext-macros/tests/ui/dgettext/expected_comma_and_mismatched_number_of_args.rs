use gettext_macros::*;

fn main() {
    dgettext!("domainname", "{}" 1, 2);
}
