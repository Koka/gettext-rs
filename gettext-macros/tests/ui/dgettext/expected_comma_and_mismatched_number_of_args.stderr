error: expected `,`
 --> tests/ui/dgettext/expected_comma_and_mismatched_number_of_args.rs:4:34
  |
4 |     dgettext!("domainname", "{}" 1, 2);
  |                                  ^

error: 1 positional argument in format string, but there are 2 arguments
 --> tests/ui/dgettext/expected_comma_and_mismatched_number_of_args.rs:4:29
  |
4 |     dgettext!("domainname", "{}" 1, 2);
  |                             ^^^^
